### ISSUE TITLE
Guaranteed First Aid in Locked Stores

### ISSUE THREAD

**Issue by Grubleafeater** [CleverRaven/Cataclysm-DDA#55055]
### Is your feature request related to a problem? Please describe.

Unlooted gun shops and similar shops that are locked up should ALWAYS have first aid kits available. It's actually required by law in New York that all small businesses have first aid kits, and, from a liability perspective, even non-small businesses have strong reasons to keep them stocked.

Not to say that they need to be particularly well stocked--alcohol wipes and gauze would probably suffice.

### Solution you would like.

Add first aid kits to gun shops and other secured shops.

**Comment by LeahLuong** [comment]
I believe lore situates this game in MA, @ least @ game start. Is this law there??

**Comment by Grubleafeater** [comment]
Looked it up--it actually looks like there's separate federal requirements, too.

https://smallbusiness.chron.com/law-require-first-aid-kit-workplace-11360.html

" To ensure the business is prepared for unexpected injury, federal regulations require businesses to maintain properly stocked first aid kits in convenient areas when an in-house medical facility is not present on the premises.

Standards
OSHA, the Occupational Safety and Health Administration, outlines the required contents of the first aid kit. A small business with three employees should maintain at least one to two of each item in the first aid kit. The business must increase the count for additional employees to ensure an ample supply. The additional count requirement is not listed in the general industry guidelines. However, an OSHA representative will provide the required count information as needed for each specific business.

Contents
The first aid kit must include small and large sterile gauze pads, adhesive bandages, gauze roller bandages, triangular bandages, wound cleaner, scissors, a blanket, adhesive tape, latex gloves, resuscitation equipment, elastic wraps, a splint and clear directions for obtaining emergency assistance."

In other words, it requires:

Gauze, bandages, adhesive bandages, scissors, a blanket(?), adhesive tape, latex, RECUSSITATION EQUIPMENT(?!), and a splint. By law.

**Comment by LeahLuong** [comment]
There is the stipulation of 3 employees. If the gun store or pawn shop is operated by less, I don't believe this req holds. I'm sure many small businesses skirt the law here & inspectors aren't bashing their doors down to hand them stiff fines.

Blankets can help in cases of severe blood loss where the body has trouble regulating its own temp. There are other cases. Resuscitation equipment might be something as simple as an endotracheal tube (for clearing an airway) or manually operated resuscitation mask (for when you don't want to [or can't] put your lips on some1 else's to breathe for them); I don't think they mean every employer needs a defibrillator. Disposable medical gloves are considered resuscitation equipment.

@Grubleafeater Good on you for digging up the relevant info but I doubt this issue is addressed anytime soon. Even if it is, it would be unrealistic for _**every**_ locked store to get a kit.

**Comment by Grubleafeater** [comment]
Doesn't say that there only needs to be one if there are at least 3 people, only that the rec is 1-2 items if 3 people.

**Comment by wapcaplet** [comment]
Off-topic, but I'm curious what OSHA has to say about the lack of fire exits or back doors in a lot of public/commercial buildings in the game, too.

**Comment by LeahLuong** [comment]
Insufficient fire exits are more of a building code violation & likely handled by another bureaucratic department, depending on jurisdiction, of course.

**Comment by Grubleafeater** [comment]
Huh--should this really be tagged as a bug and not a change that should be made?

**Comment by Night-Pryanik** [comment]
Closing as stale, since stalebot can't do it by itself.
